src/Configuration/src/CloudFoundry.ServiceBinding/PostProcessors/CloudFoundryPostProcessor.cs

```diff
@@ -12,15 +12,18 @@ internal abstract class CloudFoundryPostProcessor : IConfigurationPostProcessor
     private static readonly Regex TagsConfigurationKeyRegex =
         new("^vcap:services:[^:]+:[0-9]+:tags:[0-9]+", RegexOptions.Compiled | RegexOptions.IgnoreCase, TimeSpan.FromSeconds(1));
 
+    private static readonly Regex LabelConfigurationKeyRegex =
+        new("^vcap:services:[^:]+:[0-9]+:label+", RegexOptions.Compiled | RegexOptions.IgnoreCase, TimeSpan.FromSeconds(1));
+
     public abstract void PostProcessConfiguration(PostProcessorConfigurationProvider provider, IDictionary<string, string?> configurationData);
 
-    protected IEnumerable<string> FilterKeys(IDictionary<string, string?> configurationData, string tagValueToFind)
+    protected IEnumerable<string> FilterKeys(IDictionary<string, string?> configurationData, string tagOrLabelValueToFind)
```

Comment:
This is an interesting idea, though I'll admit some skepticism about how much of an impact it will have.

Trying it out, I realized this method for our tests was generating data that isn't actually accurate for the `p-identity` service binding, so I'll need to rework at least this part regardless of how the flags enum fits in

```
    protected Dictionary<string, string?> GetConfigurationData(string bindingType, string bindingProvider, string bindingName,
        params Tuple<string, string>[] secrets)
    {
        var dictionary = new Dictionary<string, string?>(StringComparer.OrdinalIgnoreCase)
        {
            // no way to set labels instead of tags in this method right now
            [$"vcap:services:{bindingProvider}:0:tags:0"] = bindingType,
            [$"vcap:services:{bindingProvider}:0:name"] = bindingName
        };
```